openshift/rosa
# OCM-1294 | feat: Modified password validation to use common library

[OCM-1294](https://issues.redhat.com/browse/OCM-1294)

# TEST 

- create a rosa cluster 
- Create idp on the cluster with the bellow command,
`rosa create idp -t htpasswd --name jaya --username jaya --password Forgerock@#$123 -c <cluster-name>`

Make sure it throws a more informative response on why the password doesn't meet requirements.

# Draft PR
